db/migrate/20240501173819_add_visual_editor_to_editions.rb

```diff
@@ -0,0 +1,5 @@
+class AddVisualEditorToEditions < ActiveRecord::Migration[7.1]
+  def change
+    add_column :editions, :visual_editor, :boolean
```

Comment:
Part of me still wonders whether we're going to want to show when the Visual Editor was exited at some point. I suppose we can do another migration if and when that becomes necessary